### PR TITLE
Settings-Screen-Admin-App-General

### PR DESCRIPTION
**Pull request by dejetem**
https://github.com/ALCOpenSource/Mentor-Management-System-Team-2/issues/6
Settings Screen Admin App {general}
Screenshot 2023-04-14 at 1 48 23 PM

### CODE REVIEW

File: frontend/mms-Admin/components/componentStyles/profileimage.module.css [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+.web_sub_div label,
+.cityname_div label,
+.countryname_div label,
+.div_about label,
+.fullName_div label {
+  font-size: 1.2rem;
+}
+
+.bio {
+  width: 915px;
```
Comment (emmsdan) [inline frontend/mms-Admin/components/componentStyles/profileimage.module.css:79]: this assumes, that your design is not responsive

File: frontend/mms-Admin/components/componentStyles/profileimage.module.css [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+  font-size: 1.2rem;
+  padding: 5px;
+  margin-bottom: 20px;
+  border: none;
+  border-radius: 5px;
+  box-shadow: 0px 0px 5px rgba(0, 0, 0, 0.2);
+}
+
+.site {
+  width: 915px;
```
Comment (emmsdan) [inline frontend/mms-Admin/components/componentStyles/profileimage.module.css:90]: this assumes, that your design is not responsive

File: frontend/mms-Admin/pages/api/auth/[...nextauth].js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import NextAuth from "next-auth";
+import GoogleProvider from "next-auth/providers/google";
+
+export default NextAuth({
+  providers: [
+    // Google Provider
```
Comment (emmsdan) [inline frontend/mms-Admin/pages/api/auth/[...nextauth].js:6]: remove redundant comment

File: frontend/mms-Admin/styles/admin/login.module.css [hunk 4]
```diff
@@ -1,3 +1,9 @@
+/* .main_container {
```
Comment (emmsdan) [inline frontend/mms-Admin/styles/admin/login.module.css:1]: remove unused code

**Review by emmsdan — CHANGES_REQUESTED** [review]
Please, fix code quality issues and make sure you to remove unused code.

File: frontend/mms-Admin/components/settings/ProfileImage.js [hunk 5]
```diff
@@ -0,0 +1,281 @@
...
+    linkdein: '',
+    instagram:''
+  });
+  const handleInputChange = (event) => {
+    setFormData({
+      ...formData,
+      [event.target.name]: event.target.value,
+    });
+  };
+  console.log(formData);
```
[the remark this answers is not in the document]
Comment (dejetem) [reply frontend/mms-Admin/components/settings/ProfileImage.js:33]: Okay

File: frontend/mms-Admin/components/componentStyles/sidebar.module.css [hunk 6]
```diff
@@ -0,0 +1,52 @@
+.sidebar {
+  /* height: 100vh;
```
[the remark this answers is not in the document]
Comment (dejetem) [reply frontend/mms-Admin/components/componentStyles/sidebar.module.css:2]: Okay

File: frontend/mms-Admin/components/componentStyles/profileimage.module.css (hunk 2 above)
Comment (dejetem) [reply frontend/mms-Admin/components/componentStyles/profileimage.module.css:90]: it is

File: frontend/mms-Admin/components/componentStyles/profileimage.module.css (hunk 1 above)
Comment (dejetem) [reply frontend/mms-Admin/components/componentStyles/profileimage.module.css:79]: it is

**Review by emmsdan — CHANGES_REQUESTED** [review]
Include screenshots and update design to march figma

**Review by emmsdan — CHANGES_REQUESTED** [review]
- this is does not match the design, remove shadow under text, and make sure your designs match
- the words should not be an image
- no, you don't download the text just the logos
- not responsive
-

**Comment by dejetem** [comment]
> * this is does not match the design, remove shadow under text, and make sure your designs match
> * the words should not be an image
> * no, you don't download the text just the logos
> * not responsive

Is it the whole design that does not match or only the text logo

**Comment by emmsdan** [comment]
> Is it the whole design that does not match or only the text logo

double-check your design with that of Figma.

**Comment by dejetem** [comment]
> > Is it the whole design that does not match or only the text logo
> 
> double-check your design with that of Figma. especially your alignments, colours, and sizes

Okay

**Pull request merged**